### PR TITLE
PE-707: Price calc compatibility

### PR DESCRIPTION
**Pull request by fedellen**
This PR removes the shebang from ardriveFactory and removes node-fetch. Both of these issues made Core incompatible with the price calc repo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "ardrive-core-js",
-	"version": "1.0.2",
+	"version": "1.0.3",
```
Comment (arielmelendez) [inline package.json:3]: So we're just rolling the bump into this branch/release?

Comment (fedellen) [reply package.json:3]: This branch is off `master` so I've bumped the version here to release, yes. Still in hotfix mode until price calc is working 😬

File: src/utils/common.ts [hunk 2]
```diff
@@ -492,17 +492,17 @@ export async function sanitizePath(path: string): Promise<string> {
...
-	try {
-		const res = await fetch('https://api.coingecko.com/api/v3/simple/price?ids=arweave&vs_currencies=usd');
-		usdPrice = (await res.clone().json()).arweave.usd;
-		return usdPrice;
-	} catch (err) {
-		console.log('Error getting AR/USD price from Coingecko');
-		return 0;
-	}
-}
+// export async function getArUSDPrice(): Promise<number> {
```
Comment (arielmelendez) [inline src/utils/common.ts:495]: ?

Comment (fedellen) [reply src/utils/common.ts:495]: This was using the removed package `node-fetch` and its not being used anywhere so I commented it out for now.

I think we will move the fiat oracles to core for dealing with usd price

**Review by arielmelendez — APPROVED** [review]

**Pull request merged**